src/main/java/io/vertx/core/impl/DeploymentManager.java

```diff
@@ -374,19 +377,80 @@ public void unregisterVerticleFactory(VerticleFactory factory) {
     return factoryList;
   }
 
+  private static URL mapToURL(String path) {
+    try {
+      if (path.startsWith("/")) {
```

Comment:
This will not work for windows.
Test for this regexp ^(/|[a-zA-Z]\\:\\\\).* will work for absolute paths but not for relative paths.

But how to check if the string starts with a protocol or an relative path, because at least in linux a colon is a valid char in in the path ....

Any ideas?

Reply:
Okay, I found a way to solve this problem:

The only place where this method is called with strings containing an URL is in extractCPByManifest(). I will not longer call mapToURL from this place, then I can expect always a String which represents an absolute file path, which can be handled by new File()...